src/fiber/context.cr

```diff
@@ -0,0 +1,55 @@
+class Fiber
+  # :nodoc:
+  #
+  # The arch-specific make/swapcontext assembly relies on the Context struct and
+  # expects the following layout. Avoid moving the struct properties as it would
+  # require to update all the make/swapcontext implementations.
+  @[Extern]
+  struct Context
+    property stack_top : Void*
+    property resumable : LibC::Long
```

Comment:
What about using an enum for `resumable`, like:
```crystal
property state : State
enum State
  Resumable 
  Running
  Dead
end
```

Reply:
Sadly, no, the value must be accessible from ASM, and the enumeration wouldn't do much.